packages/api/src/auth/verifiers/sha256Verifier.ts

```diff
@@ -0,0 +1,121 @@
+import { createHmac, timingSafeEqual } from 'crypto'
+
+import {
+  VerifyOptions,
+  WebhookVerificationError,
+  VERIFICATION_ERROR_MESSAGE,
+  DEFAULT_WEBHOOK_SECRET,
+} from './index'
+import type { WebhookVerifier } from './index'
+
+export interface Sha256Verifier extends WebhookVerifier {}
+
+function toNormalizedJsonString(payload: Record<string, unknown>) {
+  return JSON.stringify(payload).replace(/[^\\]\\u[\da-f]{4}/g, (s) => {
+    return s.substr(0, 3) + s.substr(3).toUpperCase()
+  })
+}
```

Comment:
Weird!

Reply:
Yeah - I took the code directly from GitHub's sign and verify:

https://github.com/octokit/webhooks-methods.js/blob/main/src/node/sign.ts#L45